res/css/_animations.scss

```diff
@@ -53,3 +53,23 @@ limitations under the License.
         transition: none;
     }
 }
+
```

Comment:
Please add those above `@media (prefers-reduced-motion)` and make those animations so we don't animate things where there's `prefers-reduced-motion` preference turned on in browser.